app/locale/en.coffee

```diff
@@ -9,7 +9,8 @@
     learn_more: "Learn more"
     classroom_in_a_box: "A classroom in-a-box for teaching computer science."
     codecombat_is: "CodeCombat is a platform <strong>for students</strong> to learn computer science while playing through a real game."
-    our_courses: "Our courses have been specifically playtested <strong>to excel in the classroom</strong>, even by teachers with little to no prior programming experience."
+    our_courses: "Our courses have been specifically playtested <strong>to excel in the classroom</strong>, even for teachers with little to no prior programming experience." #{change}
```

Comment:
Since it's just tweaking the wording without changing the meaning, I don't think this needs to propagate a review.